samples/mqtt/README.md

```diff
@@ -0,0 +1,11 @@
+MQTT samples
+
+To run the samples, you need a running MQTT broker.
+
+To run a sample MQTT broker using docker:
+
+```bash
+echo "listener 1883 0.0.0.0
+allow_anonymous true" > samples/mosquitto.conf
+docker run --rm --name mosquitto -p 1883:1883 -v "$(pwd)/samples/mosquitto.conf:/mosquitto/config/mosquitto.conf" eclipse-mosquitto
```

Comment:
looks like `mosquitto.conf` is not part of the samples here? where is this command supposed to be run (since it refs `pwd`)?

Reply:
@embano1 Sorry for the confusion. I just updated the doc with a simpler command, so that it can run any path.